test/modules/backup-gcs/backup_backend_test.go

```diff
@@ -34,7 +33,23 @@ import (
 	"github.com/weaviate/weaviate/usecases/config"
 )
 
-func Test_GCSBackend_Backup(t *testing.T) {
+var override = []string{"", ""}
```

Comment:
can we use named parameters in methods instead of this global override array? it's hard to decipher later in the code what `override[2]` means